ui-sandbox/src/pages/Home/Home.tsx

```
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import reactLogo from "../../assets/react.svg";
+import viteLogo from "/vite.svg";
+import './home.css'
+
+export function Home() {
+   const [count, setCount] = useState(0);
+  return (
+    <div>
+      <div>
+        <a href="https://vite.dev" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)